test/system/kubernetes/setupTestPod.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #
-# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Revert.

Doesn't this break the Rat check?

Reply:
Reverted.